README.md

```diff
@@ -1,19 +1,82 @@
-### thanks for 100 stars :)
-
 ```
  _   _____   __   ____  ___  ___   _  ________                
 | | / / _ | / /  / __ \/ _ \/ _ | / |/ /_  __/__________  ____
 | |/ / __ |/ /__/ /_/ / , _/ __ |/    / / / /___/ __/ _ \/ __/
 |___/_/ |_/____/\____/_/|_/_/ |_/_/|_/ /_/     /_/ / .__/\__/ 
                                                   /_/         
 ```
+[![Discord][discord-shield]][discord-url]
+[![License][license-shield]][license-url]
+[![Stars][stars-shield]][stars-url]
+[![Forks][forks-shield]][forks-url]
+[![Releases][releases-shield]][releases-url]
+[![Language][language-shield]][language-url]
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>  
+    <li><a href="demo">Demo</a></li>
+    <li><a href="installation">Installation</a></li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#support">Support</a></li>
+    <li><a href="#disclaimer">Disclaimer</a></li>
+  </ol>
 
-[![Discord](https://img.shields.io/badge/discord-join-7389D8?style=flat&logo=discord)](https://discord.gg/uGuswsZwAT)
+ 
+ 
+<!-- DEMO --> 
+## Demo
+<a>
+    <img src="assets/Demo1.png" alt="Demo" width="205" height="112">
+    <img src="assets/Demo2.png" alt="Demo" width="205" height="112">
+</a>
 
-# Installation
+ 
+ 
+<!-- INSTALLATION --> 
+## Installation
 - Download the latest [release](https://github.com/colinhartigan/valorant-rpc/releases/) and run it.

```

Comment:
Change the link to: https://github.com/colinhartigan/valorant-rpc/releases/latest/download/valorant-rpc.exe

This link automatically downloads the latest .exe